docs/products/compute/compute-instances/guides/placement-groups/index.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Work with Placement Groups"
+description: "Learn how to group your compute instances to best meet your delivery model."
+keywords: ["placement-group", "affinity", "compliance"]
+published: TBD
+modified: 2024-02-14
+modified_by:
+  name: Linode
+authors: ["Linode"]
+---
+
+When you deploy several compute instances in an Akamai data center ("region"), they're default placed wherever there's available space. But, you may want them in a specific physical location, to best support your need. Do you want them close together, even on the same host to speed up performance? Or, do you want to disperse them across several hosts to support high availability? Placement groups let you determine this physical location to meet either of these models.
```

Comment:
How about...

> When you deploy several Compute Instances in an Akamai data center (_region_), they are allocated to physical machines chosen from a variety of factors, including the instances' plans and availability for those plan sizes. It may be desirable for these instances to be stored on the same physical machine to increase performance. Or, you may wish to disperse them across several machines to support high availability. The placement groups feature grants you more control over how groups of Compute Instances are distributed within a region.